x-pack/plugins/ml/public/file_datavisualizer/components/edit_flyout/overrides.js

```diff
@@ -39,6 +42,7 @@ const quoteOptions = getQuoteOptions();
 
 const LINES_TO_SAMPLE_VALUE_MIN = 3;
 const LINES_TO_SAMPLE_VALUE_MAX = 1000000;
+const CUSTOM_DROPDOWN_OPTION = 'custom';
```

Comment:
it might be worth moving this to `options_list.js` and using it in there for `TIMESTAMP_OPTIONS` and also in `DELIMITER_OPTIONS` which currently uses `'other'`
It would be good if text for both custom override options was the same. This would require some other changes to the custom delimiter checks.

Reply:
Good call. Added in 329e1348ecf281cbaae64e57e111a24379f3c217